app/services/itt_new_starter_data_signoff_trainees_service.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class IttNewStarterDataSignoffTraineesService
```

Comment:
aside - if there is a new thing, we should call it `new-starter-sign-off` or similar - not tie it to census.

Reply:
The reason i didnt use the trainee filter is that the only parameter we care about here that has a filter is the start_academic_year being the current cycle. The draft filtering ocurrs in the policy scope before you get to the page and there is no filter to screen before a specific date. I could work the Trainees::Filter stuff in here but itd only save me a line =P. It probably would be cleaner to make a new export that simply calls the old export code, rather than passing this service into the old export though, so I'll do that =)